downloads/swan-lake-release-notes/swan-lake-alpha3/RELEASE_NOTE.md

```diff
@@ -2,41 +2,928 @@
+
+##### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha3%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
 
 #### Standard Library
-- 
+
+##### Log Package Updates
+
+###### Introduced additional log levels and log functions
+
+There are 4 log levels: `DEBUG`, `ERROR`, `INFO`, and `WARN` and their respective functions: `printDebug`, `printError`, `printInfo`, and `printWarn`. To set the global log level, place the entry given below in the `Config.toml` file:
+```toml
+[log] 
+level = "[LOG_LEVEL]"
+```
+
+Each module can also be assigned its own log level. To assign a log level to a module, provide the following entry in the `Config.toml` file:
+```toml
+[[log.modules]] 
+name = "[MODULE_NAME]" 
+level = "[LOG_LEVEL]"
+```
+
+##### OS Package Updates
+
+- Removed the `exec` function.
+
+##### Task Package Updates
+
+The module has been revamped by removing the `Scheduler` and `Listener` classes and introducing the following functions to schedule and manage the job either one-time or periodically.
+
+- Configures the scheduler worker pool with the worker count and max waiting time.
+```ballerina
+import ballerina/task;
+
+task:Error? output = task:configureWorkerPool(6, 7000);
+```
+
+- Schedules the Ballerina job at a specified time.
+```ballerina
+import ballerina/task;
+import ballerina/time;
+
+class MyJob {
+   *task:Job;
+
+   public function execute() {
+       // logic goes here
+   }
+}
+
+time:ZoneOffset zoneOffset = {hours: 5, minutes: 30};
+time:Civil time = {
+    year: 2021,
+    month: 4,
+    day: 12,
+    hour: 23,
+    minute: 20,
+    second: 50.52,
+    timeAbbrev: "Asia/Colombo",
+    utcOffset: zoneOffset
+};
+task:Error|task:JobId id = task:scheduleOneTimeJob(new MyJob(), time);
+```
+
+- Schedules the recurring Ballerina job according to the given duration.
+```ballerina
+import ballerina/task;
+
+class MyJob {
+   *task:Job;
+
+   public function execute() {
+       // logic goes here
+   }
+}
+task:Error|task:JobId id = task:scheduleJobRecurByFrequency(new MyJob(), 1);
+```
+
+- Unschedules the particular job.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:unscheduleJob(id);
+```
+
+- Pauses all the jobs.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:pauseAllJobs();
+```
+
+- Resumes all the jobs.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:resumeAllJobs();
+```
+
+- Pauses the particular job.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:pauseJob(id);
+```
+
+- Resumes the particular job.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:resumeJob(id);
+```
+
+- Gets all the running jobs.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:getRunningJobs();
```

Comment:
Is this code snippet correct? When saying `getRunningJobs()`, isn't the expectation a list of all running jobs? 🤔 @ThisaruGuruge

Reply:
@kalaiyarasiganeshalingam

Reply:
Fixed in 5b060cc

Reply:
Refer https://github.com/ballerina-platform/ballerina-dev-website/pull/1995#discussion_r598651385